core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java

```diff
@@ -311,7 +311,16 @@ public void run() {
                     while (!(shouldStop.get() || source.exhausted())) {
                         int bytesReceived = inputStream.read(buffer);
 
-                        handler.channelRead(null, Unpooled.wrappedBuffer(buffer, 0, bytesReceived));
+                        int offset = 0;
+                        for (int i = 0; i < bytesReceived; i++) {
+                            // some handlers like JsonResponseCallbackHandler do not work with multi-line buffers
+                            boolean isLineBreak = buffer[i] == '\n';
```

Comment:
This one matches any line break, but I assume only line breaks between json objects should be considered (as delimiter for [line-delimited JSON](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON)). So, what happens when something like below is part of the response? 🤔 

```
{"foo":"bar\nbaz"}
{"anything":"else"}
```

Reply:
"\\n" != "\n" :)

Reply:
oh my. gotcha! :)